01 Cloud Platform/99 API Reference/08 Optimization Management/03 Read Optimization/metadata.json

```diff
@@ -0,0 +1,12 @@
+{
+    "type": "metadata",
+    "values": {
+        "description": "Read an optimization.",
```

Comment:
Some of these descriptions are quite short. We could extend to include more context. For example, this one could be something like "Get the result of an optimization job in QuantConnect Cloud through a request with the QC API."